packages/fx-core/src/plugins/resource/bot/aadRegistration.ts

```diff
@@ -66,7 +69,7 @@ export class AADRegistration {
         }
       );
     } catch (e) {
-      throw new ProvisionError(CommonStrings.AAD_CLIENT_SECRET, e);
+      throw this.handleError(e, CreateSecretError);
```

Comment:
Since handleError is a static method, better to call it using `AADRegistration.handleError()`

Reply:
I use the handleError in this class, it is different from AADRegistration.handleError()